packages/metadata-fields/FeaturedMetadata.jsx

```diff
@@ -250,26 +251,20 @@ export default function FeaturedMetadata({ data }) {
         </div>
 
         {firstRow.length === 0 ? (
-          <hr className="su-border-none su-grow su-w-70 su-h-2 su-bg-gradient-to-r su-from-digital-red-dark su-to-digital-red-light dark:su-from-palo-verde dark:su-to-olive md:su-w-auto md:su-h-3 su-mt-38 md:su-mt-0" />
+          <div
+            aria-hidden="true"
+            className="su-border-none su-grow su-w-70 su-h-2 su-bg-gradient-to-r su-from-digital-red-dark su-to-digital-red-light dark:su-from-palo-verde dark:su-to-olive md:su-w-auto md:su-h-3 su-mt-38 md:su-mt-0"
+          />
         ) : (
           ""
         )}
       </div>
 
-      {firstRow.length > 0 ? (
-        <hr
-          aria-hidden="true"
-          className="su-border-none su-grow su-w-70 su-h-2 su-bg-gradient-to-r su-from-digital-red-light su-to-digital-red-dark dark:su-from-palo-verde dark:su-to-olive md:su-w-auto md:su-h-3 su-mt-38 md:su-mt-0"
-        />
-      ) : (
-        ""
-      )}
-
       {campus &&
       ![null, undefined, "undefined", ""].includes(campus.asset_assetid) ? (
-        <div className="su-text-center su-flex su-flex-col su-gap-20 md:su-gap-26">
+        <div className="su-text-center su-mt-34 md:su-mt-58 lg:su-mt-61 su-flex su-flex-col su-gap-20 md:su-gap-26">
```

Comment:
```suggestion
        <div className="su-text-center su-rs-mt-4 su-flex su-flex-col su-gap-20 md:su-gap-26">
```
I suspect the designer intended to use modular spacing 4 here, though the difference is that the 61px margin kicks in at 2XL breakpoint not LG (that's true for all modular spacing in Decanter - the largest value kicks in at 2XL). I'm going to let you decide whether you'd like this change or not @iamrentman 😄

Reply:
I've made this change and applied to all the other sections for consistency.